<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbulEduIdentitesUIV1</class>
 <widget class="QWidget" name="AbulEduIdentitesUIV1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fiche utilisateur</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pageIdentiteDetails">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QFrame" name="frmGauche">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0" colspan="6">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QComboBox" name="cbIdentite"/>
            </item>
            <item>
             <widget class="QPushButton" name="btnUnifier">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Vous souhaitez unifier ce compte avec un autre ?</string>
              </property>
              <property name="text">
               <string>Unifier</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0" rowspan="2">
           <widget class="AbulEduFlatBoutonV1" name="btnPhoto">
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="4" colspan="2">
           <widget class="QLineEdit" name="leNom">
            <property name="text">
             <string>Nom</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="lblPrenom">
            <property name="text">
             <string>Prénom</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4" colspan="2">
           <widget class="QLineEdit" name="lePrenom">
            <property name="text">
             <string>Prénom</string>
            </property>
           </widget>
          </item>
          <item row="6" column="4">
           <widget class="QLineEdit" name="leResponsable2">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="5">
           <widget class="QToolButton" name="btnResponsable2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="6">
           <widget class="QFrame" name="frame">
            <property name="styleSheet">
             <string notr="true">background: #fff;border-radius: 5px;</string>
            </property>
            <layout class="QGridLayout" name="layoutMessage">
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QPlainTextEdit" name="lblMessage">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>100</height>
                </size>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="5" column="0" colspan="3">
           <widget class="QLabel" name="lblResponsable">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Responsables légaux :</string>
            </property>
           </widget>
          </item>
          <item row="5" column="4">
           <widget class="QLineEdit" name="leResponsable1">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="5">
           <widget class="QToolButton" name="btnResponsable1">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="lblNom">
            <property name="text">
             <string>Nom</string>
            </property>
           </widget>
          </item>
          <item row="4" column="4" colspan="2">
           <widget class="QLineEdit" name="leDomaine">
            <property name="styleSheet">
             <string notr="true">background: #eee</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="4">
           <widget class="QLabel" name="lblEcole">
            <property name="text">
             <string>Domaine</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="4">
           <widget class="QLabel" name="lblLogin">
            <property name="text">
             <string>Identifiant</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4" colspan="2">
           <widget class="QLineEdit" name="leLogin">
            <property name="styleSheet">
             <string notr="true">background: #eee</string>
            </property>
            <property name="text">
             <string>prenom.nom@ecole</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFrame" name="frmDroite">
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QFormLayout" name="_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lblTelephone">
            <property name="text">
             <string>Téléphone</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leTelephone"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblMobile">
            <property name="text">
             <string>Mobile</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leMobile"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lblMail">
            <property name="text">
             <string>Courriel</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leMail">
            <property name="text">
             <string>prenom.nom@ryxeo.com</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lblAdresse">
            <property name="text">
             <string>Adresse</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="leAdresse1"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lblCodePostal">
            <property name="text">
             <string>Code postal</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="leAdresse2"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="lblVille">
            <property name="text">
             <string>Ville</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="leAdresse3"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="lblPassword1">
            <property name="text">
             <string>Mot de passe :</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="lePassword1">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="lblPassword2">
            <property name="text">
             <string>Vérification :</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="lePassword2">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnMesLogs">
           <property name="text">
            <string>&amp;Journal de mes activités</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnUpdate">
           <property name="text">
            <string>&amp;Mettre à jour ma fiche</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageIdentiteAvatars">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
border:none;</string>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>959</width>
            <height>504</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayoutAvatars"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnAnnuler">
           <property name="text">
            <string>&amp;Annuler</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnUpdateAvatar">
           <property name="text">
            <string>&amp;Mettre à jour mon avatar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AbulEduFlatBoutonV1</class>
   <extends>QPushButton</extends>
   <header>abuleduflatboutonv1.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbIdentite</tabstop>
  <tabstop>btnUnifier</tabstop>
  <tabstop>btnPhoto</tabstop>
  <tabstop>leNom</tabstop>
  <tabstop>lePrenom</tabstop>
  <tabstop>leLogin</tabstop>
  <tabstop>leDomaine</tabstop>
  <tabstop>leResponsable1</tabstop>
  <tabstop>btnResponsable1</tabstop>
  <tabstop>leResponsable2</tabstop>
  <tabstop>btnResponsable2</tabstop>
  <tabstop>leTelephone</tabstop>
  <tabstop>leMobile</tabstop>
  <tabstop>leMail</tabstop>
  <tabstop>leAdresse1</tabstop>
  <tabstop>leAdresse2</tabstop>
  <tabstop>leAdresse3</tabstop>
  <tabstop>lePassword1</tabstop>
  <tabstop>lePassword2</tabstop>
  <tabstop>btnMesLogs</tabstop>
  <tabstop>btnUpdate</tabstop>
  <tabstop>lblMessage</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
